src/pokemon.c

```diff
@@ -5935,13 +5935,13 @@ u16 HoennToNationalOrder(u16 hoennNum)
 
 u16 SpeciesToCryId(u16 species)
 {
-    if (species <= 250)
+    if (species <= SPECIES_CELEBI - 1)
         return species;
 
-    if (species < 276)
+    if (species < SPECIES_TREECKO - 1)
         return 200;
```

Comment:
Should be SPECIES_UNOWN - 1

Reply:
That causes it to become nonmatching.

Reply:
The `return 200` should be `return SPECIES_UNOWN - 1`.

Reply:
*Ah.* I wish Github would highlight the referenced line in yellow for these things. :V